app/components/TimeSeriesChart.tsx

```diff
@@ -122,20 +128,23 @@ export default function TimeSeriesChart({
   interpolation = 'linear',
   startTime,
   endTime,
-  maxValue,
   unit,
-}: Props) {
+}: TimeSeriesChartProps) {
+  // We use the largest data point +20% for the graph scale. !rawData doesn't
+  // mean it's empty (it will never be empty because we fill in artificial 0s at
+  // beginning and end), it means the metrics requests haven't come back yet
+  const maxY = useMemo(() => {
+    if (!rawData) return null
+    const dataMax = Math.max(...rawData.map((datum) => datum.value))
+    return roundUpToDivBy(dataMax * 1.2, TICK_COUNT) // avoid uneven ticks
+  }, [rawData])
+
   // If max value is set we normalize the graph so that
   // is the maximum, we also use our own function as recharts
-  // doesn't fill the whole domain (just upto the data max)
-  const yTicks = maxValue
-    ? {
-        domain: [0, maxValue],
-        ticks: getVerticalTicks(6, maxValue),
-      }
-    : {
-        tickSize: 6,
-      }
```

Comment:
I got rid of `tickSize` here — I thought it was weird to only specify it when there's a max, but also [6 is the default anyway](https://recharts.org/en-US/api/YAxis#tickSize) so we're not doing anything.